configs/tour_mode_choice_annotate_choosers_preprocessor.csv

```diff
@@ -72,3 +72,6 @@ local,_DF_IS_TOUR,'tour_type' in df.columns
 destination in central business district,destination_in_cbd,"(reindex(land_use.area_type, df[dest_col_name]) < setting('cbd_threshold')) * 1"
 #,,FIXME diagnostic
 #,sov_dist_rt,(odt_skims['SOV_DIST'] + dot_skims['SOV_DIST'])
+,is_dest_cordon, df[dest_col_name].isin(LIST_SMALL_CORDON_TAZS_SF)
```

Comment:
I think this looks right but I'd just confirm that the destination TAZ names and `LIST_SMALL_CORDON_TAZS_SF` are the same data type. I think this might fail if the TAZ names are strings (I think they're integers but let's make sure)